pkg/version/main.go

```diff
@@ -45,6 +45,11 @@ func (v Version) String() string {
 	)
 }
 
+// JaegerVersion returns the latest know version of jaeger by this operator
+func (v Version) JaegerVersion() string {
```

Comment:
Not sure it is necessary to create an additional functon - could get from `DefaultJaeger()` or `Get().Jaeger`.